projects/addon-doc/components/main/main.template.html

```diff
@@ -1,12 +1,26 @@
 <tui-theme-night *ngIf="theme.isDefaultTheme && night.useDefaultNightTheme && (night$ | async)"></tui-theme-night>
 <tui-root>
-    <div class="tui-doc-page">
+    <div
+        *tuiLet="(aside$ | async) || [] as asideMenus"
```

Comment:
Let's go with ngIf if it's always truthy anyway, no need to introduce another module.

Reply:
fixed